test/unit/Escrow.t.sol

```diff
@@ -47,9 +53,15 @@ contract EscrowUnitTests is UnitTest {
     Duration private _minLockAssetDuration = Durations.from(1 days);
     Duration private _maxMinAssetsLockDuration = Durations.from(100 days);
     uint256 private stethAmount = 100 ether;
```

Comment:
The name `stethSharesAmount` seems more appropriate, as it’s referred to as shares in the tests